sqltap/sqltap.py

```diff
@@ -229,13 +257,17 @@ class QueryGroup(object):
     queries, including their query text/time/count, backtrace stacks.
     """
 
+    ParamsID = 1
```

Comment:
Basically, it's a global. Every unique combination of parameters will have a different ID.